tests/ballerina-test/src/test/java/org/ballerinalang/test/nativeimpl/functions/CryptoTest.java

```diff
@@ -92,4 +97,44 @@ public void testHashing() {
         Assert.assertFalse(returnValues == null || returnValues.length == 0 || returnValues[0] == null);
         Assert.assertEquals(returnValues[0].stringValue(), expectedSHA256Hash);
     }
+
+    @Test
+    public void testCRC32ForText() {
+        String payload = "Ballerina CRC32 Hash Test";
+        String expectedCRC32Hash = "e1ad4853";
+
+        BValue[] returnValues = BRunUtil.invoke(compileResult, "testHashWithCRC32ForText",
+                                                new BValue[]{new BString(payload)});
+        Assert.assertFalse(returnValues == null || returnValues.length == 0 || returnValues[0] == null);
+        Assert.assertEquals(returnValues[0].stringValue(), expectedCRC32Hash);
+    }
+
+    @Test
```

Comment:
May be we can provide descriptions for these test annotations why these tests were added. Eg:- @Test(description = "This test following functionality etc")